src/Symfony/Component/Validator/Constraints/AtLeastOneOf.php

```diff
@@ -26,7 +26,7 @@ class AtLeastOneOf extends Composite
     ];
 
     public $constraints = [];
-    public $message = 'This value should satisfy at least one of the following constraints:';
+    public $message = 'This value should satisfy at least one of the following constraints:{{ child_messages }}';
```

Comment:
```suggestion
    public $message = 'This value should satisfy at least one of the following constraints: {{ child_messages }}';
```
It reads more natural with a whitespace in between

Reply:
Done! :heavy_check_mark: